fiftyone/utils/cvat.py

```diff
@@ -1146,9 +1145,10 @@ def _parse_anno_dict(d):
 
         attributes = []
         for attr in _ensure_list(d.get("attribute", [])):
-            name = attr["@name"].lstrip("@")
-            value = _parse_attribute(attr["#text"])
-            attributes.append(CVATAttribute(name, value))
+            if "#text" in attr:
```

Comment:
Related patch, '#text' is apparently not present when a label attribute with empty value is parsed from the XML. 

This change will just omit the attribute in that case, which is fine, since our codebase has a strict convention that None and missing are interchangeable. I considering inserting `None`, but there is other code in this module that tries to guess attribute types based on their values in some cases. Might not actually have been an issue (didn't fully investigate), but type inference would have failed on `None`, so it felt dangerous.